crates/dojo/world/src/remote/permissions.rs

```diff
@@ -97,6 +97,10 @@ impl PermissionsUpdateable for ResourceRemote {
             ResourceRemote::Namespace(namespace) => {
                 namespace.update_writer(contract_address, is_writer)
             }
+            ResourceRemote::Library(_) => {
+                // ?
```

Comment:
The behavior is the same as contract. The contract address here is the address of the writer, not the address of the library. This will then register `true/false` for the library into the world's storage.